packages/std_polyfills/README.md

```diff
@@ -0,0 +1,38 @@
+<div align="center">
+	<p>
+		<a href=""><img src="https://raw.githubusercontent.com/DiscordLuau/.github/master/resource/DiscordLuau-Banner.png" width="512" alt="discord-luau"/></a>
+	</p>
+</div>
+
+## [DiscordLuau - Std Polyfills](https://pesde.dev/packages/discord_luau/std_polyfills)
+
+DiscordLuau - Std Polyfills provides polyfills for the standard library. When transitioning Discord-Luau to a new Luau runtime, ensure the standard library is fully polyfilled.
+
+All Discord-Luau packages depend on this package as a core dependency.
+
+### Installation
+
+To use DiscordLuau State, add it to your project using the pesde package manager:
+
+```bash
+pesde add discord_luau/std_polyfills
+```
+
+### Getting Started
+
+1. Require the library in your project:
+```luau
+local Std = require("./lune_packages/std_polyfills")
```

Comment:
`Std` is not a class. It should use `camelCase`, i.e., `std`. We need to refactor the rest of this package to do this too.

Reply:
updated.